core/message_pool.go

```diff
@@ -12,6 +12,19 @@ import (
 	"github.com/filecoin-project/go-filecoin/types"
 )
 
+// MessageTimeOut is the number of tipsets we should receive before timing out messages
+const MessageTimeOut = 6
```

Comment:
Please make this a construction parameter, or even better, a parameter to the timeout/expire method.

Reply:
My reasoning for not having a public timeout/expire method are explained in my comments about the role of `MessagePool`.

If I make this a construction parameter, I'll just have to define this constant somewhere else so that I can pass it in. Right now, it's `Node` that would need to know about this constant and pass it in. This seems a bad trade.

We do need to figure what to do with all these constants we have lying around the code. They need to be accounted for in our model for change. Right now I don't know what that looks like, so, in the mean time, this is the simplest option.